panther_gpiod/src/gpio_driver.cpp

```diff
@@ -138,6 +147,16 @@ void GPIODriver::ChangePinDirection(const GPIOPin pin, const gpiod::line::direct
   gpio_info.value = line_request_->get_value(gpio_info.offset);
 }
 
+bool GPIODriver::IsPinAvaible(const GPIOPin pin) const
+{
+  for (auto & info : gpio_info_storage_) {
+    if (info.pin == pin) {
+      return true;
+    }
+  }
+  return false;
+}
+
```

Comment:
```suggestion
bool GPIODriver::IsPinAvaible(const GPIOPin pin) const
{
  return std::any_of(gpio_info_storage_.begin(), gpio_info_storage_.end(),
                     [&](const auto &info) { return info.pin == pin; });
}

```
It is considered more idiomatic in modern C++. Just another suggestion :p 

With such changes, we still need to include <algorithm>.

Reply:
I'm sorry. There is a typo in my comment caused by my Ctrl+V skills. With such changes, we still need to include <algorithm>.